test/sea/sea.js

```diff
@@ -166,7 +174,7 @@ describe('SEA', function(){
       SEA.decrypt(s, pair, function(v){
       expect({a:1}).to.eql(v);
       done();
-      });});});});});});});});});});});});});});});});});});});});});});});});});});});
+      });});});});});});});});});});});});});});});});});});});});});});});});});});});});});
     })
```

Comment:
That looks strange :P but if works, fine.

Reply:
yeah i just added another unit test so added  brackets in same format

Reply:
@mhelander any idea about the 4e2 issue?

Reply:
Dunno why this test fails. It shouldn't...

Reply:
@mhelander 
its how the data being parsed back after encryption. Is there a way to save the type of the data? because 4e2 could be a string or a scientific notation number. how are javascript types handled?